CONTRIBUTING.md

```diff
@@ -76,6 +76,13 @@ If using `pip` and `virtualenv` (not recommended), you need to `cd` to the direc
 $ pip install -e .
 ```
 
+Finally, you need to install the core dependencies in the subrepos.
+This will install `spyder`, `spyder-kernels`, `qdarkstyle`, `qtconsole`, and `python-lsp-server` in editable mode.
```

Comment:
I think `spyder` is not actually installed with the `install_subrepos.py` script, right?